doc/user_guide/user_guide.md

```diff
@@ -263,15 +265,57 @@ Please notice that we convert Avro complex types to the JSON Strings. Use Exasol
 `VARCHAR(n)` column type to store them. Depending on the size of complex type,
 set the number of characters in the VARCHAR type accordingly.
 
-## Import From Kafka Cluster
+## Importing Raw JSON
```

Comment:
Could you please move this section after the "Importing Avro Record" and before "Secure Connection"?

It is more easier in my eyes to read this way.